src/config/external-applications.md

```diff
@@ -82,3 +82,20 @@ example, to build the control package, you must install the `openblas-devel`,
 
 To use MATLAB's help browser, live scripts, add-on installer, and simulink,
 install the `libselinux` package.
+
+## Steam
+
+Steam can be installed either via a native package, which requires [enabling the
+"nonfree" repository](../xbps/repositories/index.md#nonfree), or via
+[Flatpak](##flatpak). The list of dependencies for different platforms and
+troubleshooting information for the native package can be found in its
+[Void-specific documentation](./package-documentation/index.html), while this
+section will deals with potential issues faced by Flatpak users.
+
+If you are using a different drive to store you game library, the `--filesystem`
+option from [flatpak-override(1)](https://man.voidlinux.org/flatpak-override.1)
+can prove useful.
+
+If sound is not working when Steam is installed using Flatpak,
+[PulseAudio](./media/pulseaudio.md) auto-activation might not be working
+correctly. Make sure PulseAudio is running before launching Steam.
```

Comment:
I think we can consider moving this to the flatpak item, since other flatpak applications have the same issue. What do you think?

Reply:
I think it looks better in the flatpak section and as such I have moved it there in my most recent push.